msg/SubArea.msg

```diff
@@ -0,0 +1,2 @@
+geometry_msgs/Point center_point
```

Comment:
Volgens mij zijn deze nergens stamped, m.a.w., je weet nooit t.o.v. welke frame deze punten zijn gedefinieerd. Ook dat zou ik updaten nu we toch message definities aan het aanpassen zijn. De entity pose is in "map"? Hoe zit dit met de convex hull: is dat ook map of is dat het entity frame id? Hoe zit dat met deze messages?

Reply:
Heb nu de point in de volumes wel stamped gemaakt. De rest laat ik nu ff voor wat het is. Dat is een beerput open trekken.